app/views/layouts/application.html.erb

```diff
@@ -23,23 +23,23 @@
           <%= link_to "All Items", "/items"%>
           <%= link_to "Cart: #{cart.total_items}", "/cart" %>
           <%= link_to "Profile", "/profile"%>
-          <%= link_to "Log out", "/logout"%>
+          <%= link_to "Log out", logout_path, method: "delete" %>
```

Comment:
Check this page out. Its very helpful and gave me a good lead on logout
https://3rd-edition.railstutorial.org/book/log_in_log_out